packages/fabrix/src/directive/zod.ts

```diff
@@ -1,4 +0,0 @@
-export const fallbackDefault =
-  <T>(defaultValue: T) =>
-  (value: T | null | undefined) =>
-    value ?? defaultValue;
```

Comment:
I noticed that this implementation is harmful in the way it unintentionally drops the type of schema and changes the schema to type T, so removed it.